app/controllers/SignUpController.scala

```diff
@@ -1,13 +1,17 @@
 package controllers
 
-import data.repositories.UserRepository
+import com.google.gson.Gson
+import domain.repositories.UserRepository
+import domain.requestEntities.RUser
 import javax.inject.Inject
 import play.api.mvc.{AbstractController, ControllerComponents}
 
-class SignUpController @Inject()(cc: ControllerComponents, repo: UserRepository)
+class SignUpController @Inject()(cc: ControllerComponents, repo: UserRepository, gson: Gson)
   extends AbstractController(cc){
 
   def signUp() = Action{ implicit request =>
-    Ok(repo.signUp("em", "pa").toString())
+    val json = request.body.asJson.get.toString()
```

Comment:
@Bukarachka we need to add a validation of data in order to be sure user is being passed as parameter

Reply:
should I do it as another ticket, or here?

Reply:
here